src/app.py

```diff
@@ -1,90 +1,886 @@
-    performantes, accessibles, open-source et abordables pour la protection contre les risques naturels. """)
-
-    st.sidebar.write('\n')
-    st.sidebar.markdown("""[Site Web](https://pyronear.org/)""")
-
-    # Disabling warning
-    st.set_option('deprecation.showfileUploaderEncoding', False)
-
-    # Choose your own image
-    uploaded_file = cols[0].file_uploader("Upload files", type=['png', 'jpeg', 'jpg'])
-    submitted = False
-    if uploaded_file is not None:
-        st.warning(
-            """En partageant cette photo, vous adhérez à la démarche Pyronear """
-            """de construction d'un jeu de données collaboratif."""
+    return upload
+
+
+########################################################################################################################
+# Main app -------------------------------------------------------------------------------------------------------------
+
+app = dash.Dash(
+    __name__,
+    external_stylesheets=[dbc.themes.UNITED]
+)
+server = app.server
+
+# We define a few attributes of the app object
+app.title = 'Pyronear - Crowdsourcing platform'
+app.config.suppress_callback_exceptions = True
+
+# Navbar Title
+user_item = html.Div(
+    "Surveillez les départs de feux",
+    id="user-div",
+    className="mx-auto order-0",
+    style={'color': 'white', 'align': 'center', 'justify': 'center'}
+)
+
+# Alert monitoring screen, to be displayed in CODIS
+link_to_website_button = dbc.NavLink(
+    children=[
+        html.Div(
+            children=[
+                html.I(
+                    className="mx-auto order-0",
+                ),
+                html.Span("à propos de Pyronear"),
+            ]
         )
-        img = uploaded_file.read()
-        cols[0].image(img)
-        form = cols[1].form("Information de la photo")
-        form.date_input("Date de la photo", max_value=date.today())
-        form.time_input("Heure approximative")
-        form.selectbox("Département", config.deps)
-        form.multiselect("Étiquettes", ["fumée", "flammes", "nuages", "éblouissement"])
-        submitted = form.form_submit_button("Envoyer")
+    ],
+    href="https://pyronear.org",
+    style={
+        "font-size": "17px",
+        "font-weight": "bold",
+        "color": "#054546",
+        "text-decoration": "underline"
+    }
+)
+
+left_column_width = '45%'
+form_inputs_width = '80%'
+
+app.layout = html.Div(
+    [
+        dcc.Store(id='store-user-agent-string'),
+
+        # Navbar
+        dbc.Navbar(
+            [
+                # html.A(
+                #     dbc.Row(
+                #         [
+                #             dbc.Col(html.Div('Hello')),
+                #         ],
+                #         align="center",
+                #         no_gutters=True,
+                #     ),
+                #     href="#",
+                # ),
+                dbc.NavbarToggler(id="navbar-toggler"),
```

Comment:
we don't need navigation?

Reply:
No, and this removes the useless button that we noticed during the call if I am not mistaken.